code_schemes/yes_no.json

```diff
@@ -0,0 +1,91 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Radio Questions Yes_No",
```

Comment:
Can we have a different yes_no scheme for each week? The yes/no labels are being used for different questions in each week. Having different schemes means we can't accidentally make non-sensical comparisons between weeks.

Reply:
Thanks! have created separate yes_no schemes for the respective Radio questions.